test/prism_regression/assign_to_class_variable.rb

```diff
@@ -4,14 +4,15 @@
 @@regular = 1
 
 # Compound assignment operators
-@@bitwise_and &= 2
-@@bitwise_xor ^= 4
-@@shift_right >>= 5
-@@shift_left <<= 6
-@@subtract_assign -= 7
-@@module_assign %= 8
-@@bitwise_or |= 9
-@@divide_assign /= 10
+@@bitwise_and     &= 2
```

Comment:
These spaces help test our logic for handling the `message_loc`.

THey're also just happen to be aesthetically pleasing :)